_source/logzio_collections/_log-sources/cisco-securex.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Ship Cisco SecureX data to Logz.io
+logo:
+  logofile: securex-logo.png
+  orientation: vertical
+data-source: Cisco SecureX
+data-for-product-source: Logs
+open-source:
+  - title: Cisco SecureX API fetcher
+    github-repo: logzio-api-fetcher
+contributors:
+  - nshishkin
+shipping-tags:
+  - popular
+order: 470
+---
+
+
+Cisco SecureX connects the breadth of Cisco's integrated security portfolio and your infrastructure. This integration allows you to collect data from Cisco SecureX API and send it to your Logz.io account. 
+  
+
+<div class="tasklist">
+
+
+
+##### Pull the Docker image of the Logz.io API fetcher
+
+```shell
+docker pull logzio/logzio-api-fetcher
+```
+
+
+##### Create a local directory for this integration
+
+You will need a dedicated directory to use it as mounted directory for the Docker container of the Logz.io API fetcher.
+
+```shell
+mkdir logzio-api-fetcher
+cd logzio-api-fetcher
+```
+
+##### Create a configuration file
+
+In the directory created in the previous step, create a file `config.yaml` using the example configuration below:
+
+```yaml
+logzio:
+  url: https://<<LISTENER-HOST>>:8071
+  token: <<LOG-SHIPPING-TOKEN>>
+
+auth_apis:
+  - type: cisco_secure_x
+    name: cisco
+    credentials:
+      id: <<API_CREDENTIALS_ID>>
+      key: <<API_CREDENTIALS_KEY>>
+    settings:
+      time_interval: 5
+      days_back_fetch: 7
+    filters:
+      event_type%5B%5D: '1090519054'
+      start_date: 2021-10-05T10%3A10%3A10%2B00%3A00
+    custom_fields:
+      type: cisco
+      level: high
+  - type: general
```

Comment:
Please remove the general section (from this line to the end of the configuration)